assets/scss/6-components/cta-block/_cta-block.scss

```diff
@@ -0,0 +1,44 @@
+// CTA Block (c-cta-block)
+//
+// Call to action with messaging and buttons. **The window size determines the layout.**
+//
+//
+// Markup: 6-components/cta-block/cta-block.html
+//
+// Styleguide 6.0.1
+
+$cta-block-min-col: px-to-rem(160px);
+$cta-block-btn-height: px-to-rem(29px);
+
+.c-cta-block {
+  border: 1px solid $color-white-off;
+  border-radius: $border-radius-b;
+  @supports (display: grid) {
```

Comment:
Do we need to use `@supports` for grid stuff? Given we're not providing fallback styles for browsers that don't support it, we could maybe ditch it. But I could be missing something.

Reply:
I suppose not. Removed 👍